src/audio/codec_adapter/codec/waves.c

```diff
@@ -0,0 +1,754 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Waves Audio Ltd. All rights reserved.
+//
+// Author: Oleksandr Strelchenko <oleksandr.strelchenko@waves.com>
+//
+#include <sof/audio/codec_adapter/codec/generic.h>
+#include <sof/audio/codec_adapter/codec/waves.h>
+
+#include <MaxxEffect/MaxxEffect.h>
+#include <MaxxEffect/MaxxStream.h>
+#include <MaxxEffect/MaxxStatus.h>
+#include <MaxxEffect/Initialize/MaxxEffect_Initialize.h>
+#include <MaxxEffect/Process/MaxxEffect_Process.h>
+#include <MaxxEffect/Process/MaxxEffect_Reset.h>
+#include <MaxxEffect/Control/RPC/MaxxEffect_RPC_Server.h>
+#include <MaxxEffect/Control/Direct/MaxxEffect_Revision.h>
+
+#define MAX_CONFIG_SIZE_BYTES (8192)
+#define NUM_IO_STREAMS (1)
+
```

Comment:
@stolx sorry if I havent made myself clear. 
``<>`` vs ``""`` included headers has to do with the way the headers are searched.
``""`` -  means search locally.
``<>`` - search globally in the include paths.

SOF generic headers should be included using ``<>``. Maxx specific headers should prefferably be included
using ``" "``.
e.g
```
#include <sof/audio/codec_adapter/codec/generic.h>
#include <sof/audio/codec_adapter/codec/waves.h>
```
```
 #include "MaxxEffect/MaxxEffect.h"
```
btw, where are Maxx specific headers? Are proprietary and not included in SOF repo?

Reply:
Also, when you fix something and it is reasonably correct from your side please mark the comments as resolved.

Reply:
@dbaluta 
Yes, API definition is not added to SOF repo and shared together with a library.
fixed "" vs <> issue